srtgo.go

```diff
@@ -344,16 +345,16 @@ func (s SrtSocket) PacketSize() int {
 	return s.pktSize
 }
 
-// PollTimeout - Return polling max time, in milliseconds, for connect/read/write operations.
+// PollTimeout - Return polling max time, for connect/read/write operations.
 // Only applied when socket is in non-blocking mode.
-func (s SrtSocket) PollTimeout() int64 {
-	return s.pollTimeout
+func (s SrtSocket) PollTimeout() time.Duration {
+	return time.Duration(s.pollTimeout) * time.Millisecond
```

Comment:
Even though result is exactly the same, for coherence, what do you think if we do this way?

```suggestion
	return time.Duration(s.pollTimeout * time.Millisecond)
```

Reply:
that wouldn't work because we need to cast pollTimeout to a duration first.